src/mongoc/mongoc-bulk-operation.c

```diff
@@ -219,8 +256,10 @@ mongoc_bulk_operation_update (mongoc_bulk_operation_t *bulk,
                               const bson_t            *document,
                               bool                     upsert)
 {
+   bool multi = true;
```

Comment:
It was hardcoded before so I didn't change it. I could add multi as another function parameter (just like upsert)